Marlin/src/core/language.h

```diff
@@ -162,6 +162,8 @@
 #define MSG_BEGIN_FILE_LIST                 "Begin file list"
 #define MSG_END_FILE_LIST                   "End file list"
 #define MSG_INVALID_EXTRUDER                "Invalid extruder"
+#define MSG_INVALID_E_STEPPER               "Invalid E stepper"
+#define MSG_E_STEPPER_NOT_SPECIFIED         "E stepper not specified"
```

Comment:
I'm not sure how I feel about this terminology. When we do filament change, we geeks are talking about E steppers, but for common users I think we need to use more common terms. In this case, for filament change I think we should refer to Filament 1, Filament 2, etc.

Reply:
I used that terminology to be consistent with the E_STEPPERS nomenclature in Configuration.h.

Reply:
`E_STEPPERS` is not meant to be a user-aware value, and it's actually provided by `Conditionals_LCD.h`. We do mention "mixing steppers" in the configs, however. Anyway, since these messages are pertinent to filament change only —and only with mixing— it seems more proper to refer to the "filament index" or to even _slightly_ misuse the term "Extruder" in order to stay close to the established convention.

Reply:
Ahh. I meant to reference `MIXING_STEPPERS` instead of `E_STEPPERS`. Personally, I dislike the use of the term extruder, since that really referrs to the combination of the nozzle and the associated feeder(s), which is really confusing in mixing and MMU setups. Perhaps we could use the term "feeder"?

Reply:
Hmm. "Feeder" is really the best term. For filament change, the convention of "Filament 1" "Filament 2" etc. seems like it would be the least jarring.